es_service_updater.py

```diff
@@ -17,53 +20,56 @@ def lambda_handler(event, context):
     if 'SLACK_CHANNEL' in os.environ:
         slack_channel = os.environ['SLACK_CHANNEL']
 
-    if 'DOMAIN_NAME' in os.environ:
-        domain_name = os.environ['DOMAIN_NAME']
+    if 'DOMAIN_NAME_LIST' in os.environ:
+        domain_name_list = os.environ['DOMAIN_NAME_LIST']
 
     if 'REGION' in os.environ:
         region = os.environ['REGION']
```

Comment:
If we are running one of these per region - you could infer the region from the region that the lambda is running in...